inductiva/fluids/scenarios/wind_tunnel/wind_tunnel.py

```diff
@@ -86,13 +104,16 @@ def simulate(self,
         """Simulates the wind tunnel scenario synchronously.
 
         Args:
+            simulator: Simulators available [OpenFOAM].
```

Comment:
Give a description of the parameter, e.g. "Simulator used to simulate the scenario. Valid simulators: OpenFOAM."